test/extended/testdata/router/router-grpc-interop.yaml

```diff
@@ -129,190 +25,30 @@ objects:
       app: grpc-interop
   spec:
     containers:
-    - image: golang:1.14
+    - image: ${IMAGE}
       name: server
-      command: ["/workdir/grpc-server"]
-      env:
-      - name: GRPC_GO_LOG_VERBOSITY_LEVEL
-        value: "99"
-      - name: GRPC_GO_LOG_SEVERITY_LEVEL
-        value: "info"
-      - name: GODEBUG
-        value: http2debug=1
-      readinessProbe:
-        httpGet:
-          path: /healthz
-          port: 8080
-        initialDelaySeconds: 10
-        periodSeconds: 3
+      command: ["ingress-operator", "serve-grpc-test-server"]
```

Comment:
There is another subcommand `serve-http2-test-server` in router-http2.yaml -- are there 2 new subcommands?

Reply:
Yes, I see there are.

Reply:
These are already in cmd aren't they?

Reply:
There are two new commands, one serves the tests in http2.go and the the is for the grpc-interop tests.